imgparse/imgparse.py

```diff
@@ -803,6 +831,18 @@ def get_unique_id(image_path, xmp_data=None):
     :param xmp_data: used internally for memoization. Not necessary to supply.
     :return: unique image id in the form <session_id>_<image_id>
     """
+    # DJI CaptureUUID
+    make, model = get_make_and_model(image_path)
+    if make == "DJI":
+        try:
+            return str(xmp_data["drone-dji:CaptureUUID"])
+        except KeyError:
+            logger.warning(
+                "Couldn't determine unique id. Parsing image ID from file name."
+            )
+            img_name_split = os.path.basename(image_path).split("_")
+            return "_".join(img_name_split[0:3])
```

Comment:
DJI image names include timestamp and image number, if CaptureUUID isn't included then this should work as a back-up for a unique ID.

`DJI_20241011132945_0007_MS_G_50213_11384_22895_67114.TIF`

Reply:
What is this function used for? I've never seen it

Reply:
What specifically is this function used for? Why can't you just fall back to the full filename? I see below is the weird regex expression just to parse the numbers from the image. e.g. `IMG_12345` -> `12345`. IMO the final fallback could be just `return os.path.splitext(os.path.basename(image_path))[0]` so it works with any sensor